public/_data/repositories/delivery.yml

```
@@ -0,0 +1,32 @@
+section:
+  description: Tools to ensure your applications have access to the privileged systems they need.
+  categories:
+  - name: Secretless Broker
+    description: Secure your apps by making them Secretless.
```

```suggestion
    description: Securely connect your applications to services they need - without ever having to fetch or manage passwords and keys.
```
I like the secretless.io tag line better.  WDYT?

---

yup, agree. I regret not at least nailing this one! :P